coordinatorlib/src/main/resources/com.socrata.datacoordinator.truth.schema/migrate.xml

```diff
@@ -34,6 +34,7 @@
     <include file="com.socrata.datacoordinator.truth.schema/20171020-migrate-geo-secondary-cookie-pk-to-sid.xml"/>
     <include file="com.socrata.datacoordinator.truth.schema/20171122-add-secondary-move-jobs-table.xml"/>
     <include file="com.socrata.datacoordinator.truth.schema/20171128-add-collocation-manifest.xml"/>
+    <include file="com.socrata.datacoordinator.truth.schema/20180110-change-collocation-manifest.xml"/>
```

Comment:
Any reason why it needs to blow the whole table away and not just rename columns?

Reply:
Because I thought it was easier to do it that way... not real reason